buildSrc/src/main/kotlin/toolkit-testing.gradle.kts

```diff
@@ -0,0 +1,68 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import software.aws.toolkits.gradle.ciOnly
+
+val mockitoVersion: String by project
+val mockitoKotlinVersion: String by project
+val assertjVersion: String by project
+
+plugins {
+    id("java") // Need for jacoco
+    id("jacoco")
+    id("org.gradle.test-retry")
+    id("com.adarshr.test-logger")
+}
+
+dependencies {
+    testImplementation("com.nhaarman.mockitokotlin2:mockito-kotlin:$mockitoKotlinVersion")
+    testImplementation("org.mockito:mockito-core:$mockitoVersion")
+    testImplementation("org.assertj:assertj-core:$assertjVersion")
+
+    // We don't dictate junit vs testng vs junit5 here since it's not consistent
```

Comment:
nit: "since it's not consistent" -> "Since we use both junit 4 and 5"?

Reply:
and testng

Reply:
reworked comment